src/visGeometry/index.ts

```diff
@@ -726,11 +724,6 @@ class VisGeometry {
         this.updateScene(this.currentSceneAgents);
     }
 
-    public setColorChangeByIds(ids: number[]): void {
-        this.colorChangeIds = ids;
-        this.updateScene(this.currentSceneAgents);
```

Comment:
this updateScene call is needed to trigger the canvas to re-render but we didn't need to store the ids of the agents that were just changed color, we have that information elsewhere